Functional Monitoring documentation update (5 of 5) (#54)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/afm-view-schedules-private-location.adoc

:imagesdir: ../assets/images

> You can determine the number of schedules that use a private location. 

Reviewer: Ah, at first I thought that "determine" was the ability to configure rather than view....

Reviewer: How about...:
```suggestion
View the number of schedules that use a private location by navigating to that location.
```
??? Isn't this kind of gratuitous?

Reviewer: Yeah...this topic could be one sentence somewhere else. Something to look at when you really get into the weeds with this legacy stuff.

When initially specifying the private location, a test run worker associated with the schedule is created in that location. Viewing the location shows how many workers are running.

== Procedure

. Display the list of locations by opening *Location Manager* from the homepage.

> . Tap the *Private Locations* tab.

Reviewer: click
